src/promptflow/promptflow/_sdk/entities/_flow.py

```diff
@@ -361,3 +370,33 @@ def invoke(self, inputs: dict) -> "LineResult":
                 data=inputs,
             )
             return result
+
+
+class AsyncProtectedFlow(ProtectedFlow):
+    """This class is used to represent an async flow."""
+
+    async def __call__(self, *async_args, **async_kwargs):
```

Comment:
async_args -> args? 
why need to emphasize the async here?

Reply:
OK